src/aap_eda/core/migrations/0040_remove_rulebookprocesslog_line_number.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 4.2.7 on 2024-05-28 21:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0039_alter_project_options"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
@jshimkus-rh Have you check with @lgalis if there is an impact on the UI?

Reply:
Have checked with others and we're almost certainly ok.
But, as I said in slack, if people want to wait on the merge until @lgalis is back and can weigh in that's fine by me.  But I'd like reviews to take place under the assumption that it'll be fine so we're ready to merge.

Reply:
LGTM, I'm not going to approve until be sure it can be merged :)